Makefile

```diff
@@ -86,14 +119,27 @@ ifndef WINDOWS
 	    echo '$(NULL_ENTROPY_WARNING)'
 endif
 
-clean:
+clean: clean_more_on_top
 	$(MAKE) -C library clean
 	$(MAKE) -C programs clean
 	$(MAKE) -C tests clean
+
+clean_more_on_top:
 ifndef WINDOWS
 	find . \( -name \*.gcno -o -name \*.gcda -o -name \*.info \) -exec rm {} +
 endif
 
+neat: clean_more_on_top
```

Comment:
`make neat` is close to what [GNU](https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html#Standard-Targets) calls `maintainer-clean`, but not exactly the same. It may be argued to be the same as `distclean` instead. Should I use my “custom” name `neat` to avoid confusion, or a more standard name to improve discoverability?

Reply:
I like `neat`. If we're not sure the meaning is the same as `maintainer-clean` or `distclean` then it's probably better to have a distinct name. I don't think discoverability matters that much considering it's only for people using the development branch as opposed to releases.